RUNNING_TESTS.md

```diff
@@ -12,24 +12,18 @@ It is possible to use [Visual Studio Community edition](https://www.visualstudio
 
 ## Building
 
-### Copy `Local.props.example` and Edit
+Simply run
 
-First copy `Local.props.example` to `Local.props` and edit it if needed, specifying whether
-you intend to build the client on Mono and what .NET version should be targeted.
+    build.sh|bat
 
-### Build
-
-Then, build in Visual Studio or build from the command line. To do the latter,
-simply run
-
-    msbuild.exe
+This will complete the code AMQP spec generation and build all projects. After this open the solution in Visual Studio.
 
 
 ## Running Tests
 
 Tests can be run from Visual Studio using [NUnit Test Adapter](https://visualstudiogallery.msdn.microsoft.com/6ab922d0-21c0-4f06-ab5f-4ecd1fe7175d).
 Note that it may take some time for the adapter to discover tests in the assemblies.
 
-Running the tests from the command line is also straightforward: use
+Running the tests from the command line is also straightforward you don't need to build first to do this: use
 
-    msbuild.exe /t:RunUnitTests projects/client/Unit/RabbitMQ.Client.Unit.csproj
+    ./run-test.sh|bat
```

Comment:
Same as above, I'd word it to make it possible to copy the command (and make it unambiguous to those new to cross-platform and/or *nix development).